app/assets/stylesheets/ginlovers.css.scss

```diff
@@ -336,7 +336,40 @@ z-index:12;
 position: fixed;
 }
 
+#load{
+    width:100%;
+    height:100%;
+    position:fixed;
+    z-index:9999;
+    background:url("https://www.creditmutuel.fr/cmne/fr/banques/webservices/nswr/images/loading.gif") no-repeat center center rgba(0,0,0,0.25)
```

Comment:
Can this image be loaded locally instead of externally? What happens if creditmutuel.fr goes offline?